pinot-servicemanager/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM apachepinot/pinot:0.5.0-SNAPSHOT-31d4194a2-20200731 as pinot
+
+WORKDIR /pinot
+COPY pinot-admin.sh bin/pinot-admin.sh
+COPY etc/* etc/
+
+# Cherry-pick files we need from Pinot from their image
+RUN cd /opt/pinot && \
+  tar -cpf - lib/* \
+  plugins/pinot-input-format/pinot-confluent-avro \
+  plugins/pinot-stream-ingestion/pinot-kafka-2.0 | \
+  (cd /pinot; tar -xpf -)
+
+# Prepage changes over base Java image
+FROM hypertrace/java:11
+LABEL MAINTAINER Hypertrace "https://www.hypertrace.org/"
+
+ENV PINOT_HOME=/opt/pinot
+ENV PINOT_VAR=/var/pinot
+ENV JAVA_OPTS="-Xms128M -Xmx192M -XX:MaxDirectMemorySize=64M -XX:+ExitOnOutOfMemoryError"
+
+WORKDIR ${PINOT_HOME}
+
+# Setup pinot user so that the image doesn't run as root
+RUN adduser -g '' -h ${PINOT_HOME} -D pinot && \
+    mkdir -p ${PINOT_VAR}/server/data ${PINOT_VAR}/controller/data && \
+    chown -R pinot ${PINOT_VAR}
+
+USER pinot
```

Comment:
Declare the `USER` above and use it in all the commands.

Reply:
done.

Reply:
@buchi-busireddy circleci builds are failing after this change:
```
Step 15/19 : COPY --from=pinot --chown=${USER} /pinot ${PINOT_HOME}
unable to convert uid/gid chown string to host mapping: can't find uid for user ${USER}: no such user: ${USER}
```
circleci uses docker version `17.09.0-ce` in their build environment while this is fixed in docker version `19.0.3.0`.
https://github.com/moby/moby/issues/35018